fcrepo-exts/fcrepo-upgrade-utils
# add support for direct and indirect containers

**JIRA Ticket**: https://jira.lyrasis.org/browse/FCREPO-3427

# What does this Pull Request do?

Adds support for migrating direct and indirect containers to F6. This PR does very little apart from no longer rejecting direct and indirect containers. The migration only works when membership triples are emitted, as implemented in [this PR](https://github.com/fcrepo-exts/fcrepo-import-export/pull/152).

# How should this be tested?

Migrate direct and indirect containers and start F6 on top of the migrated content. Unfortunately, F6 does not currently return membership triples, so it's impossible to verify that it's working conclusively.

# Interested parties
@fcrepo4/committers
